src/lib/gui/window_glfw.h

```diff
@@ -59,11 +61,8 @@ class window_glfw : public window_interface<window_glfw>
 
 		if (_display)
 		{
-			cv::Mat rgb;
-			cv::cvtColor(img, rgb, cv::COLOR_BGR2RGBA);
-			GLuint texture = cimbar::mat_to_gl::create_gl_texture(rgb);
-			_display->draw(texture);
-			glDeleteTextures(1, &texture);
+			cimbar::mat_to_gl::load_gl_texture(_texid, img);
```

Comment:
no `cvtColor` -- we're hoping for RGB by default. There will be a followup PR...

Also: now calling `glGenTextures` in the constructor, and reusing it.